src/table/toolbar/table-toolbar.component.ts

```diff
@@ -42,6 +42,7 @@ import { I18n, Overridable } from "../../i18n/i18n.module";
 	template: `
 	<section class="bx--table-toolbar">
 		<div
+			*ngIf="showToolbarActions"
```

Comment:
What if we toggle it based on the existence of the model? If there's no model then there's no need to show the actions at all ... 🤔